config/jobs/cert-manager/cert-manager/cert-manager-periodics.yaml

```diff
@@ -113,17 +110,14 @@ periodics:
     preset-ginkgo-skip-default: "true"
   spec:
     containers:
-      # TODO: change to a custom image that embeds the system tools we
-      # need (jq, make, bash, Go, etc) but without Bazel. Tracked at
-      # https://github.com/cert-manager/cert-manager/issues/4939.
-      - image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20210831-4cf7b0b-4.2.1
+      - image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20220512-b6ea825-4.2.1
         args:
-          - runner
-          - bash
-          - -c
-          - |
-            apt-get install jq -y >/dev/null
-            make -j vendor-go e2e-ci K8S_VERSION=1.19
+        - runner
+        - make
+        - -j
+        - vendor-go
+        - e2e-ci
+        - K8S_VERSION=1.19
```

Comment:
I think it might make sense to pass kube version as an env var for consistency as that's how other configurable options (i.e `GINKGO_FOCUS`/`GINKGO_SKIP`) are passed
But that might be a job for another PR as I think the make scripts currently don't read them from env (?)

Reply:
Yeah that'd make sense, definitely something to keep in mind!